### PR TITLE
PurchseOrder V3 API documentation changes for new fields

### PR DESCRIPTION
**Pull request by harishrconcur**
As part of V3 Purchase Order we introduced some changes. It is already live in Sep SU, this is to update the customer facing documentation.

Related development Jira - INV-28161

### CODE REVIEW

**Review by prithvi-varla — APPROVED** [review]

**Comment by retrosight** [comment]
Test feedback from an account with Write privileges -- please ignore.

File: src/api-reference/invoice/v3.purchase-order.markdown [hunk 1]
```diff
@@ -225,18 +233,21 @@ Name|Type|Format|Description
...
 `UnitPrice`|`string`|-|**Required** The price of each item of the line item.
+
 `VatAmount`|`string`|-|This field has not been implemented by Purchase Request yet. Any data in this field will be ignored.
 `VatRate`|`string`|-|This field has not been implemented by Purchase Request yet. Any data in this field will be ignored.
 
 #### <a name="allocation"></a>Allocation  
 
 Name|Type|Format|Description
----|---|---|---
+---|---|---|---The purchase request number.
```
Comment (sarraloew) [inline src/api-reference/invoice/v3.purchase-order.markdown]: This change broke the table.

File: src/api-reference/invoice/v3.purchase-order.markdown [hunk 2]
```diff
@@ -225,18 +233,21 @@ Name|Type|Format|Description
 `RequestedDeliveryDate`|`string`|-|The date the line item of the purchase order instructed the vendor to deliver the goods. Format: YYYY-MM-DD
 `SupplierPartID`|`string`|-|Any number that might help to identify the line item. This could be a value such as the vendor's part number or even the manufacturer number.
 `Tax`|`string`|-|Any tax that is associated with the line item.
+`UnitOfMeasureCode`|`string`|-|The unit of measure code of the line item.
 `UnitPrice`|`string`|-|**Required** The price of each item of the line item.
+
```
Comment (sarraloew) [inline src/api-reference/invoice/v3.purchase-order.markdown]: This change broke the table.

**Review by sarraloew — CHANGES_REQUESTED** [review]
The table formatting went a bit off with these changes. Also, on line 25, you have "Please use CES for batch updates." CES is an outdated term. Please replace it with the product name, which I'm guessing in this case is Concur Invoice. Thanks!

**Comment by harishrconcur** [comment]
@sarraloew Thanks for catch I missed it. I have fixed your comments in 1c46fd24

**Review by sarraloew — APPROVED** [review]

**Pull request merged**